assets/title.scss

```diff
@@ -120,7 +120,7 @@ table.table-dark th {
 }
 
 .section h2 {
-  padding-top:10rem;
+  padding-top:5rem;

```

Comment:
I haven't setup Jekyll so I didn't test that properly (I edited the styles manually in DevTools), and I trust you the style changes are fine.